turbinia/workers/artifact.py

```diff
@@ -77,6 +77,12 @@ def run(self, evidence, result):
 
     # Path to the source image/directory.
     cmd.append(evidence.local_path)
+    if not evidence.local_path:
+      result.log('Tried to run image_export without local_path')
+      result.close(
+          self, False, 'image_export.py failed for artifact {0:s}.'.format(
```

Comment:
Can we include something about the local_path not being there in the status message as well?  Maybe just append the text that was in the above log line to the status as well?  That way it surfaces in the client easier and we don't have to look at the logs to see what happened.